### DISCUSSION TITLE
fully move to a "goat" organization

### DISCUSSION THREAD

**Discussion by amtoine** [a2n-s/dotfiles#19] [General]
as you might have seen, i've opened a bunch of issues related to the shift to the "goatfiles" organization
their main goals are (**1**) to update the `README`s and (**2**) to fix all the old mentions to "amtoine" and "a2n-s", both in the code and in the _GitHub_'s metadata

## the issues
:red_circle: nothing has been done yet
:orange_circle: the conversation is _pending_
:white_circle: a draft PR has been opened
:yellow_circle: a PR review is in progress
:green_circle: the PR is closed and completed

below is the list of all these issues, to keep track of each one of them:

### :green_circle: fix the headers
- https://github.com/goatfiles/dotfiles/issues/2 :green_circle: #22 
- https://github.com/goatfiles/scripts/issues/18 :green_circle: https://github.com/goatfiles/scripts/pull/36

### :green_circle: fix the names of things
- https://github.com/goatfiles/scripts/issues/33
  - :green_circle: https://github.com/goatfiles/scripts/pull/38
  - :green_circle: #23
- https://github.com/goatfiles/pkgbuilds/issues/21 :green_circle: https://github.com/goatfiles/pkgbuilds/pull/24
- https://github.com/goatfiles/pkgbuilds/issues/23 :green_circle: https://github.com/goatfiles/pkgbuilds/pull/25

### :green_circle: fix the `README`s  
- https://github.com/goatfiles/wallpapers/issues/5 :green_circle: https://github.com/goatfiles/wallpapers/pull/6
- https://github.com/goatfiles/scripts/issues/32 :green_circle: https://github.com/goatfiles/scripts/pull/35
- https://github.com/goatfiles/icons/issues/12 :green_circle: https://github.com/goatfiles/icons/pull/13
- https://github.com/goatfiles/junnunkarim-wallpapers/issues/1 :green_circle: https://github.com/goatfiles/junnunkarim-wallpapers/pull/2
- https://github.com/goatfiles/sounds/issues/3 :green_circle: https://github.com/goatfiles/sounds/pull/4
- https://github.com/goatfiles/applications/issues/3 :green_circle: https://github.com/goatfiles/applications/pull/5

### :red_circle: repos that do not fit the organization 
- https://github.com/goatfiles/dmscripts/issues/1 :red_circle:
- https://github.com/goatfiles/polybar-themes/issues/1 :red_circle:
- https://github.com/goatfiles/neovim/issues/1 :red_circle:

**Comment by dolphinau** [comment]
Great work!
How should we proceed now?
Resolving the issues one by one? Which one first?

**Reply by amtoine** [reply]
> How should we proceed now?
>
i think the priority now is to 
- (**1**) solve the two opened PRs of [goatfiles/dotfiles](https://github.com/goatfiles/dotfiles), see [moving forward with new features](https://github.com/orgs/goatfiles/discussions/20)
  - https://github.com/goatfiles/dotfiles/pull/13
  - https://github.com/goatfiles/dotfiles/pull/15
- (**2**) get the gist of the three pinned discussions of the organization
  - [**fully move to a "goat" organization**](https://github.com/orgs/goatfiles/discussions/19): it's right here basically and lists all the opened important issues of the organization
  - [**moving forward with new features**](https://github.com/orgs/goatfiles/discussions/20): lists the current feature requests and an idea for the future
  - [**current state of the dotfiles**](https://github.com/orgs/goatfiles/discussions/18): focuses on the `README` of the [goatfiles/dotfiles](https://github.com/goatfiles/dotfiles) and the headers and deprecated mentions
- (**3**) ask me any question :wink: 

> Resolving the issues one by one? Which one first?
>
once the PRs are closed, the gist have been captured and the questions have been asked and answered, i.e. once you're up to speed :muscle:, we'll tackle these issues one a the time :smirk: 

i would say, in order of important in the four sections above:
- "fix the name of things"
- "fix the `README`s"
- "fix the headers"
- "repos that do not fit the organization"

**Reply by amtoine** [reply]
i'll have a start on these issues :+1:

**Reply by amtoine** [reply]
a bunch of new PRs to address the issues :wink:

**Reply by amtoine** [reply]
only [dotfiles#2](https://github.com/goatfiles/dotfiles/issues/2) is missing, but i'll do that tomorrow probably :+1:
